### DISCUSSION TITLE
v4.8.2

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [adminerevo/adminerevo#30] [Announcements]
We are proud to announce the birth of our first version with AdminerEvo identity, v4.8.2 of the Adminer tool.
Thanks to all who contributed to this release, published this July 21st 2023.

We will handle communication with the community ONLY through github discussions : feature requests / errors report / ideas / comments are all welcome there.

When reporting a problem, to help us investigate properly, may we ask you to enclose:
- a detailed explanation of the problem
- a scenario how to reproduce
- a list of plugins / theme installed on your setup
- if possible, transmit your install and required part(s) of the database to reproduce

We hope that this new version will give you confidence in the future of the tool and that many of you will follow and collaborate.

Regards,
L.

**Comment by nitodico** [comment]
Awesome... project looks promising!

**Reply by LionelLaffineur** [reply]
Thanks!

**Comment by victor53134** [comment]
The greatest of successes to this project and I hope there are many collaborators so that it is not a one man project.
Good luck in this challenge.
